lib/chunked-exec.js

```diff
@@ -0,0 +1,13 @@
+const {promisify} = require('util');
+const {execFile} = require('child_process');
+
+const pExecFile = promisify(execFile);
+
+const maxPaths = 1000;
+
+module.exports = async (binary, paths) => {
+	for (let group = 0; paths.length > group; group += maxPaths) {
+		// eslint-disable-next-line no-await-in-loop
+		await pExecFile(binary, paths.slice(group, group + maxPaths));
+	}
```

Comment:
Alternatively I could use one of the [many chunked array modules](https://npms.io/search?q=chunk+array) but I didn't since none of them are yours _and_ probably this shouldn't be parallelized any more than it is (200-1000 paths at a time)

I think the correct solution would be to sum the strings’ lengths, but maybe this is good enough.

Reply:
I think this is fine. Summing the strings would probably lead to too much concurrency.